app/frontend/actions/voting.tsx

```diff
@@ -54,21 +56,25 @@ export default (store: Store) => {
     state = getState()
 
     const stakePubKey = xpub2pub(
-      Buffer.from(getSourceAccountInfo(state).stakingXpub.xpubHex, 'hex')
+      Buffer.from(getSourceAccountInfo(state).stakingXpub?.xpubHex || '', 'hex')
```

Comment:
I'd rather put here a runtime assertion that this xpubHex exists - if it wouldn't, this code would break unpredictably